src/PrestaShopBundle/Bridge/AdminController/ControllerConfigurationFactory.php

```diff
@@ -64,15 +64,15 @@ public function create(
         string $controllerNameLegacy,
         string $table
     ): ControllerConfiguration {
-        $configuratorController = new ControllerConfiguration();
+        $controllerConfiguration = new ControllerConfiguration();
+        $controllerConfiguration->id = $id;
+        $controllerConfiguration->controllerName = $controllerName;
+        $controllerConfiguration->php_self = $controllerName;
+        $controllerConfiguration->controllerNameLegacy = $controllerNameLegacy;
```

Comment:
@tegbessou I don't remember the difference between `controllerName/php_self` and `controllerNameLegacy` can you remind me?

By looking at this code:

```php
        $controllerConfiguration = $this->getControllerConfigurationFactory()->create(
            $tabId,
            get_class($this),
            $legacyControllerName,
            $this->getTableName()
        );
```

I can deduce that the controller name seems to contain the Symfony controller class name, and `controllerNameLegacy` contains the legacy controller name which is useful for permissions (among other things). But do we really need the Symfony controller name and class? I would naively think that only the legacy controller name is actually useful and relevant no? (at least for hooks, modules, and stuff)

Reply:
might make sense, Im seeing most of php_self usages in front controllers only (and that is actually expected to be legacy controller than new class indeed), I wonder if we really need it at all in admin controller. Waiting for @tegbessou answer.

Reply:
Yes, we need it for hooks; when php_self is not set, hooks crash. I think instead of `get_class($this)` we can put the name of the bridge object I think